internal/api/v1/servicebindings.go

```diff
@@ -98,7 +100,7 @@ func (hc ServicebindingsController) Create(w http.ResponseWriter, r *http.Reques
 	}
 
 	for _, service := range theServices {
-		err = app.Bind(ctx, service)
+		err = wl.Bind(ctx, service)
```

Comment:
A service is bound to an application, not a specific workload. Even if the application didn't have a workload, we should still be able to bind the service to the application. As soon as a workload appears, it will have the service details mounted just because the Service was bound to the App. I don't mean we need to implement this functionality now but the change in the narrative is confusing.